fixtures/questions/create_questions_fixtures.py

```diff
@@ -1,18 +1,42 @@
+import datetime
+
+start_date = (
+  datetime.datetime.now().replace(microsecond=0) + datetime.timedelta(days=1)
+).isoformat()
+end_date = (
+  datetime.datetime.now().replace(microsecond=0) + datetime.timedelta(days=8)
+).isoformat()
+wrong_start_date = (
+  datetime.datetime.now().replace(microsecond=0) + datetime.timedelta(days=-1)
+).isoformat()
+wrong_end_date = (
+  datetime.datetime.now().replace(microsecond=0) + datetime.timedelta(days=1)
+).isoformat()
+
+
 create_question_query = '''
- mutation{
+ mutation{{
   createQuestion(questionType:"Rate",
   question:"How will you rate the brightness of the room",
-  startDate:"20 Nov 2018", endDate:"28 Nov 2018") {
-    question{
+  startDate:"{}", endDate:"{}") {{
```

Comment:
Since this is just a test fixture, I recommend that you just type the values the way they are supposed to be. e.g startDate: 2018-12-23T23:42:43. Avoid using string format unless it is very necessary. Also have a sync with the front end team to agree on the date format they will be sending to the backend api in their request.

Reply:
I have consulted with a frontend team member and, going through an implemented date feature, we have been able to come up with the DateTime format used in this PR.
With regards to typing in the date, the implementation of the feature is such that one can not set a start date before the current date and an end date before a specified period of days. The dates entered need to be dynamic for the tests to pass.

Reply:
Okay.